.azure/gpu-tests-lite.yml

```diff
@@ -0,0 +1,126 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+  tags:
+    include:
+      - '*'
+  branches:
+    include:
+      - "master"
+      - "release/*"
+      - "refs/tags/*"
+
+pr:
+  - "master"
+  - "release/*"
+
+variables:
+  - name: continue
+    value: '1'
+
+jobs:
+  - job: testing
+    strategy:
+      matrix:
+        'PyTorch - stable':
+          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.12-cuda11.6.1"
+    # how long to run the job before automatically cancelling
+    timeoutInMinutes: "80"
+    # how much time to give 'run always even if cancelled tasks' before stopping them
+    cancelTimeoutInMinutes: "2"
+    pool: azure-jirka-spot
+    container:
+      image: $(image)
+      # default shm size is 64m. Increase it to avoid:
+      # 'Error while creating shared memory: unhandled system error, NCCL version 2.7.8'
+      options: "--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=all --shm-size=512m"
+    workspace:
+      clean: all
+
+    steps:
+
+    - bash: |
+        CHANGED_FILES=$(git diff --name-status origin/master -- . | awk  '{print $2}')
```

Comment:
This job does not need this business because we are not going to mark it as required for now. Do you want me to push a commit removing it?

Reply:
Yes, feel free to. I just wanted to avoid running it on app prs because it takes away GPU resources in the queue.